tools/cdk-build-tools/package.json

```
@@ -34,29 +34,28 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/fs-extra": "^9.0.12",
+    "@types/fs-extra": "^8.1.2",
     "@types/jest": "^26.0.24",
-    "@types/semver": "^7.3.7",
-    "@types/yargs": "^16.0.1",
-    "eslint-plugin-license-header": "^0.2.0",
+    "@types/semver": "^7.3.8",
+    "@types/yargs": "^15.0.14",
     "pkglint": "0.37.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.28.4",
-    "@typescript-eslint/parser": "^4.28.4",
-    "awslint": "1.111.0",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.3.0",
```

I believe this should be `^4.33.0` based on:

https://github.com/aws/aws-cdk/blob/v1.128.0/tools/%40aws-cdk/cdk-build-tools/package.json#L46

```suggestion
    "@typescript-eslint/parser": "^4.33.0",
```

---

Fixed